### ISSUE TITLE
tedge config get should not write log messages to standard output

### ISSUE THREAD

**Issue by reubenmiller** [thin-edge/thin-edge.io#1721]
**Describe the bug**

`tedge config get <key>` produces output which is difficult to parse as it mixes log messages and configuration values.

For example, the when trying to get the current value of the `c8y.url` configuration value, when no value has been set, a warning is produced, however programmatically this is hard to parse as there is no difference between a log message saying there is no value set, and a set value.

```sh
tedge config get c8y.url
```

Produces the following on standard output (stdout)

```sh
The provided config key: 'c8y.url' is not set
```

If the log/user messages are separated on different output streams (e.g. stdout and stderr), it makes it easier to check whether a value is set or not. For example:

```sh
EXISTING_VALUE=$(tedge config get c8y.url)
if [ -n "$EXISTING_VALUE" ]; then
    echo "c8y.url has already been set"
else
    echo "c8y.url has not been set yet, setting now"
    sudo tedge config set c8y.url "$C8Y_HOST"
fi
```

**Notes**

* you can confirm if it is writing to stdout by using `tedge config get c8y.url >/dev/null`, if you don't see the same message then it is writing to stdout.

**To Reproduce**

1. Open a shell in your terminal (where tedge is installed)

2. Clear any set value for `c8y.url`

    ```sh
    sudo tedge config unset c8y.url
    ```

3. Try to get the value for `c8y.url`

    This is a simple test case, which will print out whether the test case passes or fails.

    ```sh
    [ "$(tedge config get c8y.url)" = "" ] && echo "TEST PASS: message was written to stderr :)" || echo "TEST FAIL: message was incorrectly written to stdout instead of stderr"
    ```

**Expected behavior**

* Log/user messages should be written to standard error (stderr)
* If no value is set, then no output should be written to standard output (stdout)

**Screenshots**

**Environment (please complete the following information):**
 - OS [incl. version]: Raspbian bookworm
 - Hardware [incl. revision]: armv7l (armhf)
 - System-Architecture [e.g. result of "uname -a"]: Linux raspberrypi 5.15.84-v7l+ #1613 SMP Thu Jan 5 12:01:26 GMT 2023 armv7l GNU/Linux
 - thin-edge.io version [e.g. 0.1.0]: 0.9.0

**Additional context**

The current workaround is to use `tedge config list` and grep for the key you want, however this is unnecessarily complex in comparison.

```sh
EXISTING_VALUE=$(tedge config list | grep "^c8y.url=" || true)
if [ -n "$EXISTING_VALUE" ]; then
    echo "c8y.url has already been set"
else
    echo "c8y.url has not been set yet, setting now"
    sudo tedge config set c8y.url "$C8Y_HOST"
fi
```

**Comment by Bravo555** [comment]
Will fix

**Comment by gligorisaev** [comment]
Tested and confirmed
